www/install/insertBaseConf.sql

```diff
@@ -1175,6 +1175,7 @@ INSERT INTO timezone (`timezone_name`, `timezone_offset`, `timezone_dst_offset`)
                         ('Asia/Vientiane', '+07:00', '+07:00'),
                         ('Asia/Vladivostok', '+10:00', '+10:00'),
                         ('Asia/Yakutsk', '+09:00', '+09:00'),
+                        ('Asia/Yangon', '+6:30', '+6:30'),
```

Comment:
is it possible to format the time to '+06:30'  to be more consistent with other timezone ?

Reply:
these values are not displayed. But indeed. I'll correct it